pkg/apis/aerogear/v1alpha1/types.go

```diff
@@ -62,14 +62,18 @@ type KeycloakIdentityProvider struct {
 	DisplayName               string            `json:"displayName,omitempty"`
 	InternalID                string            `json:"internalId,omitempty"`
 	ProviderID                string            `json:"providerId,omitempty"`
-	Enabled                   bool              `json:"enabled,omitempty"`
-	TrustEmail                bool              `json:"trustEmail,omitempty"`
-	StoreToken                bool              `json:"storeToken,omitempty"`
-	AddReadTokenRoleOnCreate  bool              `json:"addReadTokenRoleOnCreate,omitempty"`
+	Enabled                   bool              `json:"enabled"`
+	TrustEmail                bool              `json:"trustEmail"`
+	StoreToken                bool              `json:"storeToken"`
+	AddReadTokenRoleOnCreate  bool              `json:"addReadTokenRoleOnCreate"`
```

Comment:
Removed the "omitempty" from bool fields.

If a value was set to "false" in the spec for any of these, "omitempty" would consider that empty and not include it i.e. You could't set anything to false in the spec.